src/lib/Chip/Chip.svelte

```diff
@@ -1,10 +1,13 @@
 <script lang="ts">
-  import type { Variants } from './types'
+  type Variants = 'default' | 'info' |  'success' | 'warning' | 'danger';
 
   export let variant: Variants;
```

Comment:
Since this is the only usage of `Variants`, we could inline the type, _e.g._, 

```ts
  export let variant: 'default' | 'info' | 'success' | 'warning' | 'danger';
```